Assets/_Scripts/Systems/PlayerSystem.cs

```diff
@@ -131,9 +131,10 @@ private async void SpawnPlayer(NetworkRunner runner, PlayerRef playerRef)
 		{
 			while (playerSpawnPoints.Length == 0)
 			{
-				playerSpawnPoints = FindObjectsOfType<PlayerSpawnLocation>();
 				await Task.Delay(10);
```

Comment:
En fait, le delay ici il servait à quoi? o.O

Reply:
Aucune idée. Une relique d'un temps ou le set to spawn était appelé avant la fin du loading. Je me suis dit que c'était plus efficace par contre d'attendre avant de try get.